docs: create "add a network" post (#738)
devopness/devopness: docs/networks/index.md

---
draft: true
title: Networks

> intro: In this section you will learn how to provision and manage your networks/virtual private cloud (VPC)

Reviewer: ```suggestion
intro: Learn how to provision and manage cloud networks, also known as Virtual Private Clouds (VPCs)
```
